aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -271,13 +271,11 @@ pub fn serialize(value: &u128) -> Vec<u8> {
     bcs::to_bytes(value).expect("unexpected serialization error in aggregator")
 }
 
-// Helper for tests, #[cfg(test)] doesn't work for cross-crate.
-pub fn delta_sub(v: u128, limit: u128) -> DeltaOp {
```

Comment:
cfg(test) works, but needs to be something like  #[cfg(any(test, feature = "testing"))] then the other crate imports the feature testing